<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{
	
Background: #89CFF0;
}

QTextEdit{
Background: #ffffff;
border-radius:15px;


}

QPushButton{
Background: #ffffff;
border-radius:15px;


}


#frame_2{

Background: #ffffff;
border-radius:15px;


}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>871</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="getName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Invoice</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>571</width>
       <height>641</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Invoice</string>
      </property>
     </widget>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>531</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="phone">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>531</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="item">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>531</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Item:</string>
      </property>
     </widget>
     <widget class="QLabel" name="amount">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>531</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>AMount:</string>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="getPhone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="getItem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="getAmount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1267</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
